src/Microsoft.Extensions.Hosting.Abstractions/IHostedService.cs

```diff
@@ -1,22 +1,20 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-namespace Microsoft.AspNetCore.Hosting
+namespace Microsoft.Extensions.Hosting
 {
     /// <summary>
     /// Defines methods for objects that are managed by the host.
     /// </summary>
     public interface IHostedService
     {
         /// <summary>
-        /// Triggered when the application host has fully started and the server is waiting
-        /// for requests.
+        /// Triggered when the application host has fully started.
         /// </summary>
         void Start();
```

Comment:
There are plans for a non-web host and builder, but there will not be a shared abstraction because of the significant differences in the environments.